example/centos/pipeline/image/s390x.yaml

```diff
@@ -0,0 +1,59 @@
+name: image
+build: name:build
+stages:
+  otk.op.join:
+    values:
+      - otk.external.otk-make-partition-stages:
+          ${filesystem}
+      - - type: org.osbuild.copy
+          inputs:
+            root-tree:
+              type: org.osbuild.tree
+              origin: org.osbuild.pipeline
+              references:
+                - name:os
+          options:
+            paths:
+              - from: input://root-tree/
+                to: mount://-/
+          devices:
+            otk.op.join:
+              values:
+                - ${fs_options.devices}
+                # TBD this seems strange:
+                - disk:
+                    type: 'org.osbuild.loopback'
+                    options:
+                      filename: 'disk.img'
+          mounts:
+            ${fs_options.mounts}
+        - type: org.osbuild.zipl.inst
+          options:
+            kernel: 8-2.fk1.s390x
```

Comment:
This should use something like `${kernel.package.version}-${kernel.package.release}.${kernel.package.arch}`, please check the grub2 fragement

Reply:
Thanks!
I also assumed this is variable is the correct "location":
`location: ${fs_options.devices.boot.options.start}`